site/_docs/pages.md

```diff
@@ -95,6 +95,6 @@ simple and it works. In the end, the decision is yours!
     matter variable. In the example above, using the first method, you can
     get URL <code>http://example.com/other</code> for the file
```

Comment:
I suppose it should be there too--I just forget to change that, but browsers redirect `/other` → `/other/` automatically, so it still does work.